### DISCUSSION TITLE
[MAS-Piano] Twinkle Twinkle Little Star (DL)

### DISCUSSION THREAD

**Discussion by briaryoung** [Monika-After-Story/MonikaModDev#7788] [general - piano notes]
Hello! This was my first time coding and I would like to thank @multimokia for helping me fix the invalid code. It was a really great learning experience! I plan to program other songs that Monika sings into the piano, such as "All I Have To Do Is Dream." I hope you enjoy this simple song xD

[Twinkle Twinkle Little Star - Copy.txt](https://github.com/Monika-After-Story/MonikaModDev/files/3417829/Twinkle.Twinkle.Little.Star.-.Copy.txt)

  To download this file, find the folder called piano_songs and put the .txt file in there. Then, change the file to a .json file. Make sure you have disabled the setting, "Hide extensions for known file types." (You can't change the file type without disabling this first x3) You can find a guide for disabling this option [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/All-Features-%5BSPOILERS%5D#gifting-guide) in the **Gifts** category.

_Guide:_
TTOOPPO
IIUUYYT
OOIIUUY
OOIIUUY
TTOOPPO
IIUUYYT

**Post Script:**
  Sorry about not making this a pull request. I'm unsure of where I should put it - I totally can make a pull request if you'd like, though.

**Comment by briaryoung** [comment]
**When you tried to make a pull request but for whatever reason, it turned out as an issue instead**

**Comment by ghost** [comment]
> **When you tried to make a pull request but for whatever reason, it turned out as an issue instead**

To make pull request -> https://github.com/Monika-After-Story/MonikaModDev/pulls
Then 'New Pull Request'.  For piano songs choose  'Community' Branch.
But you need to fork MAS to make pull reqs.
There more detailed -> https://github.com/Monika-After-Story/MonikaModDev/wiki/Community-Piano-Songs#on-github

**Comment by multimokia** [comment]
Fyi:
We're probably going to keep these as issues, seeing as the community branch doesn't really get updated.

**Comment by briaryoung** [comment]
Thank you! qwq And okie, I can keep these as issues

**Comment by ThePotatoGuy** [comment]
I've updated the [Community Piano Songs](https://github.com/Monika-After-Story/MonikaModDev/wiki/Community-Piano-Songs) wiki page. We're going to compile piano jsons as a separate download, similar to spritepacks, so submitting each one as an individual issue is the right way to go for now.

**Comment by MiguelFitz** [comment]
> Hello! This was my first time coding and I would like to thank @multimokia for helping me fix the invalid code. It was a really great learning experience! I plan to program other songs that Monika sings into the piano, such as "All I Have To Do Is Dream." I hope you enjoy this simple song xD
> 
> [Twinkle Twinkle Little Star - Copy.txt](https://github.com/Monika-After-Story/MonikaModDev/files/3417829/Twinkle.Twinkle.Little.Star.-.Copy.txt)
> 
> To download this file, find the folder called piano_songs and put the .txt file in there. Then, change the file to a .json file. Make sure you have disabled the setting, "Hide extensions for known file types." (You can't change the file type without disabling this first x3) You can find a guide for disabling this option [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/All-Features-%5BSPOILERS%5D#gifting-guide) in the **Gifts** category.
> 
> _Guide:_ TTOOPPO IIUUYYT OOIIUUY OOIIUUY TTOOPPO IIUUYYT
> 
> **Post Script:** Sorry about not making this a pull request. I'm unsure of where I should put it - I totally can make a pull request if you'd like, though.

im making a submod to explain a simple way to make the json format and uses this for it, i hope you dont mind, it isnt public yet, so im gonna ask for your permission to do this, also thanks for the help
